### DISCUSSION TITLE
SCXML-to-C converter

### DISCUSSION THREAD

**Discussion by dzimmanck** [kiishor/UML-State-Machine-in-C#35] [General]
FYI, I am working on a SCXML-to-C converter written in Python that is designed to work with this library.  The idea is you can design your state machine in SCXML using an SCXML editor and than "compile" the scaffolding code for this library.

The code is not functional yet but is getting close.  You can find it at:

https://github.com/dzimmanck/scxml_to_c

**Comment by kiishor** [comment]
Hi dzimmanck,

Thanks for considering my state machine framework for your project. Your project SCXML-to-C converter is really exciting and I'm looking forward to trying it. Let me know if you need any kind of help with your project.

Regards,
Nandkishor
